sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/ConfiguredApiClientTest.java

```diff
@@ -0,0 +1,659 @@
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.apache.http.HttpStatus.SC_SEE_OTHER;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.entry;
+import static org.assertj.core.groups.Tuple.tuple;
+import static org.sdase.commons.client.jersey.error.ClientErrorUtil.convertExceptions;
+import static org.sdase.commons.client.jersey.test.util.ClientRequestExceptionConditions.processingError;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import com.github.tomakehurst.wiremock.jetty9.JettyHttpServerFactory;
+import com.github.tomakehurst.wiremock.junit.WireMockClassRule;
+import com.github.tomakehurst.wiremock.matching.RequestPatternBuilder;
+import com.github.tomakehurst.wiremock.matching.StringValuePattern;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.assertj.core.util.Lists;
+import org.glassfish.jersey.media.multipart.FormDataMultiPart;
+import org.glassfish.jersey.media.multipart.MultiPartFeature;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.sdase.commons.client.jersey.error.ClientErrorUtil;
+import org.sdase.commons.client.jersey.error.ClientRequestException;
+import org.sdase.commons.client.jersey.test.ClientTestApp;
+import org.sdase.commons.client.jersey.test.ClientTestConfig;
+import org.sdase.commons.client.jersey.test.MockApiClient;
+import org.sdase.commons.client.jersey.test.MockApiClient.Car;
+import org.sdase.commons.server.testing.EnvironmentRule;
+import org.sdase.commons.shared.api.error.ApiException;
+
+public class ConfiguredApiClientTest {
+
+  @ClassRule
+  public static final WireMockClassRule WIRE =
+      new WireMockClassRule(
+          wireMockConfig().dynamicPort().httpServerFactory(new JettyHttpServerFactory()));
+
+  public static final Car LIGHT_BLUE_CAR =
+      new Car().setSign("HH XY 4321").setColor("light blue"); // NOSONAR
+  private static final ObjectMapper OM = new ObjectMapper();
+  private static final Car BRIGHT_BLUE_CAR =
+      new Car().setSign("HH XX 1234").setColor("bright blue"); // NOSONAR
+  private final DropwizardAppRule<ClientTestConfig> dw =
+      new DropwizardAppRule<>(ClientTestApp.class, resourceFilePath("test-config.yaml"));
+
+  @Rule
+  public final RuleChain rule =
```

Comment:
Can you update this file with the latest changes in sda-dropwizard-commons? Looks like it misses some changes that were done to improve the test stability (see #628).